content/blog/2020-01-themis-0-1-0/index.Rmarkdown

```diff
@@ -0,0 +1,104 @@
+---
+title: themis 0.1.0
+author: Emil Hvitfeldt
+date: '2020-01-13'
+slug: themis-0-1-0
+description: 
+  themis 0.1.0 is now available on CRAN. Provides additional steps for `recipes`
```

Comment:
```suggestion
  themis 0.1.0 is now available on CRAN. Provides additional steps for recipes
```

Reply:
Sorry, I lied, one more change: we don't use backticks around package names. Just caught this one when looking at the actual preview.